pkg/core/localize/locales/en/cmd/context.en.toml

```diff
@@ -55,6 +55,36 @@ $ rhoas context use --name dev
 [context.use.successMessage]
 one='Current context set to "{{.Name}}"'
 
+[context.unset.cmd]
+
+[context.unset.cmd.shortDescription]
+one='Unset services in context'
+
+[context.unset.cmd.longDescription]
+one='''
+Unset services in context
+
+When you unset a service in context, it will no longer point to any instance of that service.
+'''
+
+[context.unset.cmd.example]
+one='''
+# Unset services for current context
+$ rhoas context unset --services kafka,service-registry
```

Comment:
Need feedback on the intuivity of the implementation.

Reply:
It is the same as status command so it is good. Sometimes we might use array string in cobra etc.